community/gallery/collection/02_completions.mdx

```diff
@@ -77,6 +77,13 @@ zi ice lucid wait as'completion' blockf has'alacritty'
 zi snippet https://github.com/alacritty/alacritty/blob/master/extra/completions/_alacritty
 ```
 
+### COMP: [ajeetdsouza/zoxide](https://github.com/ajeetdsouza/zoxide/blob/main/contrib/completions/_zoxide) {#comp-zoxide-completions}
```

Comment:
Wasn't sure what `{#comp-zoxide-completions}` does, so I did not add it for `delta`.